server/async_tcp.go

```diff
@@ -26,23 +29,21 @@ type AsyncServer struct {
 	multiplexer            iomultiplexer.IOMultiplexer
 	multiplexerPollTimeout time.Duration
 	connectedClients       map[int]*core.Client
-	store                  *core.Store
 	queryWatcher           *core.QueryWatcher
-	lastCronExecTime       time.Time
-	cronFrequency          time.Duration
+	shardManager           *core.ShardManager
+	ioChan                 chan *core.StoreResponse
```

Comment:
Please add a comment that suggest today the server is acting as a worker today. This will help others understand the context.

Reply:
Done